### ISSUE TITLE
Some games cause the emulator and phone to freeze and have to force a restart of the phone.

### ISSUE THREAD

**Issue by brujo5** [yuzu-emu/yuzu#12330]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

android-v156

### Description of Issue

Some games cause the emulator and phone to freeze and have to force a restart of the phone.
for example this simple game causes my phone to freeze on this screen

![Screenshot_20231210-043913_yuzu](https://github.com/yuzu-emu/yuzu/assets/4172898/3581d866-cb77-4193-8350-0f0de32ca31f)

### Expected Behavior

that shouldn't happen

### Reproduction Steps

just run the game

### Log File

[yuzu_log.txt.old.txt](https://github.com/yuzu-emu/yuzu/files/13627573/yuzu_log.txt.old.txt)

### System Configuration

CPU:sd870
GPU/Driver: Adreno 650/turnip drivers/Qualcomm driver
RAM:8gb
OS: android 13

**Comment by brujo5** [comment]
If I use dynarmic it doesn't freeze but previously the game also worked with NCE but in these latest versions it no longer works and phone freeze

**Comment by pokannce** [comment]
Yes, I have experienced this problem in the latest build too

Device - Galaxy S20 12 Gb RAM
Snapdragon 865 
Adreno 650
latest turnip version.

**Comment by ThepuncheXD7** [comment]
Me too
Device OnePlus 7t 
Snapdragon 855+
Adreno 640

**Comment by liamwhite** [comment]
If there is a regression point in the emulator, please identify the version so we can track it.

As for the contents of this issue, yuzu should not be able to make a device inoperable until reboot unless there is a programming error in the operating system. If forcibly killing yuzu does not work, this is probably an issue with the GPU driver.

**Comment by brujo5** [comment]
I just saw a pr that possibly solves this problem.  maybe it's a bug with applet resources

![Screenshot_20231210-161045_1](https://github.com/yuzu-emu/yuzu/assets/4172898/7eaebd42-649f-4b5d-a90d-4288088d9981)

**Comment by brujo5** [comment]
New log

[yuzu_log.txt.old.txt](https://github.com/yuzu-emu/yuzu/files/13630084/yuzu_log.txt.old.txt)

**Comment by liamwhite** [comment]
> If there is a regression point in the emulator, please identify the version so we can track it.

To be clear, I'm asking about which version this started happening in.

**Comment by pokannce** [comment]
For me, ig

> > If there is a regression point in the emulator, please identify the version so we can track it.
> 
> To be clear, I'm asking about which version this started happening in.

For me, It started from version 155 it was running good on all other versions.

**Comment by liamwhite** [comment]
And you have confirmed this by loading 154 and finding that it performed fine?

**Comment by brujo5** [comment]
> And you have confirmed this by loading 154 and finding that it performed fine?

I already know what causes freezes in some games, it is from using 0.75x res with NCE, if I use dynarmic I can use 0.75x res without problems with the same game. 
so I don't know if it's a bug or not since lowering the resolution is a hack and obviously will have consequences in some cases but then I don't know why dynarmic doesn't have a problem. 
v157 is fine at least in my case now, let others confirm

**Comment by brujo5** [comment]
Those who have problems with some games make sure the resolution is at 1x to see if you still have problems.

**Comment by pokannce** [comment]
> Reference

Yes I have been testing github version from 136 and till 154 I never had any crash problem, right now I have done a clean install of 157 havent tested any games, I will test it and let you know.

**Comment by pokannce** [comment]
All these problem started after I loaded the stupid game digimon story cybersleuth, at first the game starts by loading building shaders and takes lot of time, then crash. stupid game.
